pytorch_blade/src/compiler/tensorrt/tensorrt_engine_context.cpp

```diff
@@ -343,6 +341,9 @@ torch::List<torch::Tensor> TRTContext::PreProcessInputs(
     // add to cuda_ctu_inputs, make the ctu_tensor has lifetime out of the loop
     cuda_ctu_inputs.push_back(ctu_tensor);
   }
+
+  TORCH_CHECK(CheckCurrentDevice(ctu_inputs));
```

Comment:
I think you meant `cuda_ctu_tensor`. Why do we need to move the check from the beginning to the end?

Reply:
Thanks to remind me.